src/stapi_fastapi/routers/product_router.py

```diff
@@ -59,6 +59,14 @@ def __init__(
             summary="Get constraints for the product",
         )
 
+        self.add_api_route(
+            path="/order-parameters",
+            endpoint=self.get_product_order_parameters,
+            name=f"{self.root_router.name}:{self.product.id}:get-order-parameters",
+            methods=["GET"],
```

Comment:
We need to be setting the `response_model` here, similar to how we do for `/opportunities` above. This means we'll need to do some generics shenanigans like we did with `Opportunity`/`OpportunitiyCollection`, but what that ends up looking like I don't know at the moment. We'll have to play around with that to come up with the right answer.